### PR TITLE
BUG-9368: timeout shown with 2 mins

### PR DESCRIPTION
**Pull request by lcart97**
Investigate timeout issues:

2 issues:
- When timeout shows - Should be 2 minutes not 1.

- Timeout is showing at 13 minutes even when active

### CODE REVIEW

**Review by Carl-M-H — APPROVED** [review]
APPROVED! Messaged in teams chat to remind everyone about Unit Testing.

File: src/components/AppComponents/TimeoutPopup/index.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default function TimeoutPopup(props) {
 
   const initialTimeoutState = {
     countdownStart: false,
-    timeRemaining: 60,
+    timeRemaining: 120,
```
Comment (akshaykalaskar1) [inline src/components/AppComponents/TimeoutPopup/index.tsx:29]: This is a common code, is this change required in all journeys?

File: src/components/AppComponents/TimeoutPopup/index.tsx [hunk 2]
```diff
@@ -48,12 +48,12 @@ export default function TimeoutPopup(props) {
   useEffect(() => {
     if (!show) {
       // Reset countdown and related states if show is false
-      dispatch({ type: 'UPDATE_TIME_REMAINING', payload: 60 });
+      dispatch({ type: 'UPDATE_TIME_REMAINING', payload: 120 });
       dispatch({ type: 'UPDATE_SCREEN_READER_COUNTDOWN', payload: '' });
```
Comment (nickwhm) [inline src/components/AppComponents/TimeoutPopup/index.tsx:52]: Changing these will effect all journeys and the specific requirements are that we only start the visual countdown at 1 minute, and also that we update the screen readers also at that point. So if we need to change this it should be an enhancement not a bug.

Comment (lcart97) [reply src/components/AppComponents/TimeoutPopup/index.tsx:52]: Ah right, so it is meant to show at 1 minute? 

The ticket mentions that it should show at 2 minutes.

Comment (monalisasarkar123) [reply src/components/AppComponents/TimeoutPopup/index.tsx:52]: Let us know if you need any information or demo, happy to help :)

**Comment by lcart97** [comment]
Closing PR - Different approach needed.
